stdlib/CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ option(SWIFT_ENABLE_COMPATIBILITY_OVERRIDES
   "Support back-deploying compatibility fixes for newer apps running on older runtimes."
   TRUE)
 
+option(SWIFT_STDLIB_USE_NONATOMIC_RC
+    "Build the standard libraries and overlays with nonatomic reference count operations enabled"
+    FALSE)
+
+option(SWIFT_STDLIB_SINGLETHREADED_RUNTIME
+    "Build the standard libraries assuming that they will be used in an environment with only a single thread. If set, also forces SWIFT_STDLIB_USE_NONATOMIC_RC to On."
+    FALSE)
+
+if(SWIFT_STDLIB_SINGLETHREADED_RUNTIME)
+  set(SWIFT_STDLIB_USE_NONATOMIC_RC TRUE)
+endif()
```

Comment:
I'm not suggesting that you change this, simply pointing out that there exists an alternative which requires that the user enable the option which is enforced via `cmake_dependent_option`.  What you have currently is fine, I'm just trying to get others better versed in CMake.

Reply:
Good to know! I'll keep this as is, if you're okay with that, and keep `cmake_dependent_option` in mind.